src/gevent/monkey.py

```diff
@@ -951,9 +964,15 @@ def patch_signal():
     _patch_module("signal")
 
 
+def _get_patch_all_state():
+    key = '_gevent_saved_patch_all'
+    return saved.get(key)
+
+
 def _check_repatching(**module_settings):
     _warnings = []
     key = '_gevent_saved_patch_all'
+    module_settings.update(module_settings['kwargs'])
```

Comment:
It's not clear to me what difference this makes. Could you clarify please? It may be missing a test case, or perhaps that's the reason for the broken test.

Reply:
Oh. Silly me. I just reviewed #1447 and see where the problem was mentioned. So this either is the reason for the test failure, or it definitely needs a specific test.